test/SignalR.Orleans.Tests/OrleansHubLifetimeManagerTests.cs

```diff
@@ -209,9 +218,9 @@ public OrleansHubLifetimeManagerTests(OrleansFixture fixture)
 
                 await manager1.OnConnectedAsync(connection);
 
-                await manager1.AddGroupAsync(connection.ConnectionId, "name");
+                await manager1.AddGroupAsync(connection.ConnectionId, "tupac");

```

Comment:
Since we are doing integration tests and we are creating `TestFixture` and start. Tests can conflict with each other.
I've tried to move `TestFixture` per in each test but unfortunately, tests were taking so long that they were timing out. so for now, I've renamed all group tests to be unique.